deploy/roles/deploy.builder/tasks/main.yml

```diff
@@ -6,7 +6,14 @@
     - operators/prometheus/operator.yaml
     - operators/qdrouterd/operator.yaml
     - operators/smartgateway/operator.yaml
+    - operators/elasticsearch/01-service-account.yaml
```

Comment:
I should probably rename all these files to better match our existing file names. There is really no reason for ElasticSearch manifest names to be significantly different.

Reply:
I disagree; having the same names is what made it easy to figure out that you'd ripped them straight out of the upstream source.

Reply:
Cool, I will leave them alone then!